{
  "Header": {
    "Original": "CVE-2018-14353 (An issue was discovered in Mutt before 1.10.1 and NeoMutt before 2018- ...)",
    "Line": 301214,
    "ID": "CVE-2018-14353",
    "Description": "(An issue was discovered in Mutt before 1.10.1 and NeoMutt before 2018- ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-4277-1 DLA-1455-1}",
      "Line": 301215,
      "Type": "xref",
      "Bugs": [
        "DSA-4277-1",
        "DLA-1455-1"
      ]
    },
    {
      "Original": "- neomutt 20180716+dfsg.1-1 (bug #904021)",
      "Line": 301216,
      "Type": "package",
      "Package": "neomutt",
      "Kind": "fixed",
      "Version": "20180716+dfsg.1-1",
      "BugNo": 904021
    },
    {
      "Original": "- mutt 1.10.1-1 (bug #904051)",
      "Line": 301217,
      "Type": "package",
      "Package": "mutt",
      "Kind": "fixed",
      "Version": "1.10.1-1",
      "BugNo": 904051
    },
    {
      "Original": "NOTE: https://github.com/neomutt/neomutt/commit/65d64a5b60a4a3883f2cd799d92c6091d8854f23",
      "Line": 301218,
      "Type": "NOTE",
      "Description": "https://github.com/neomutt/neomutt/commit/65d64a5b60a4a3883f2cd799d92c6091d8854f23"
    },
    {
      "Original": "NOTE: https://gitlab.com/muttmua/mutt/commit/e0131852c6059107939893016c8ff56b6e42865d",
      "Line": 301219,
      "Type": "NOTE",
      "Description": "https://gitlab.com/muttmua/mutt/commit/e0131852c6059107939893016c8ff56b6e42865d"
    }
  ]
}